src/commands/flutter-symbols/upload.ts

```diff
@@ -0,0 +1,472 @@
+// Disabling no-unused-variable / no-empty temporarily so certain private methods
+// can remain in while this feature is in progress
+// tslint:disable: no-unused-variable
```

Comment:
Can we remove `tslint:disable: no-unused-variable` now? If not maybe edit the comment

Reply:
Yes they can! Good catch!